src/OpenLoco/Localisation/StringIds.h

```diff
@@ -1768,4 +1768,5 @@ namespace OpenLoco::StringIds
     constexpr string_id shortcut_game_speed_normal = 2238;
     constexpr string_id shortcut_game_speed_fast_forward = 2239;
     constexpr string_id shortcut_game_speed_extra_fast_forward = 2240;
+    constexpr string_id capacity_stringid = 2241;
```

Comment:
Could you clarify that this string is for vehicles?

Reply:
done, and I agree with your comment re. there are many nearly identical stringids, for now I can't do too much about it, but yeah in future we should have a look at cleaning them up and simplifying/de-duplicating them.